src/apps/loyalty/containers/inquiries/index.tsx

```diff
@@ -91,16 +95,14 @@ export class Inquiries extends React.Component<RelayProps, State> {
   }
 
   onArtworkSelected(conversationId: string, selected: boolean) {
-    const selectedArtworks = this.state.selected_artworks
-    selectedArtworks[conversationId] = selected
-
     this.setState({
-      selected_artworks: selectedArtworks,
+      selected_artworks: Object.assign({}, this.state.selected_artworks, { [conversationId]: selected }),
```

Comment:
Just making sure that in-place editing of `this.state.selected_artworks` doesn’t lead to side-effects (now or in the future).